src/value.h

```diff
@@ -163,9 +163,18 @@ bool SIValue_IsTrue(SIValue v);
 const char *SIType_ToString(SIType t);
 
 // Prints all individual types represented by 't', multiple types are separated by comma,
-// to a given buffer with length (bufferLen)
+// to a given buffer with length (bufferLen) last representation is preceeding by 'or' when more than 1 type
 void SIType_ToMultipleTypeString(SIType t, char *buf, size_t bufferLen);
 
+// Prints all individual types represented by 't', multiple types are separated by seperator,
+// to a given buffer with length (bufferLen), do not treat the last type differently
+// Return the number of bytes written in this call
+size_t SIType_ToMultipleTypeStringSimple(SIType t, char seperator, char *buf, size_t bufferLen);
+
+// format into buf the signature of a function with return type 'ret' and argument types 'args'
+// the format of the result is: 'name: t1 ... tn -> tn+1'
+void SITypes_SignatureToString(const char * fName, SIType ret, SIType *args, char *buf, size_t bufferLen);
+
```

Comment:
This function shouldn't be part of `value.h` as its dedicated to the functions procedure.

Reply:
done